e2e/test/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -1698,47 +1698,26 @@ describe("scenarios > dashboard > entity id support", () => {
   });
 
   it("when loading `/dashboard/entity/${non existing entity id}`, it should show a 404 page", () => {
-    const invalidSlug = "x".repeat(21);
-    cy.visit(`/dashboard/entity/${invalidSlug}`);
+    const nonExistingEntityId = "x".repeat(21);
```

Comment:
Removed reference to slugs as they're not something this endpoint should support anymore